DMTN-101.tex

```diff
@@ -29,6 +29,8 @@
 \setDocChangeRecord{%
   \addtohist{1}{YYYY-MM-DD}{Unreleased.}{Robert Lupton}
   \addtohist{1.1}{2022-08-05}{Added PTC section}{Andrés A. Plazas Malagón}
+  \addtohist{1.2}{2022-09-27}{Added gain test: 6.2}{Andrés A. Plazas Malagón}
+  \addtohist{1.2.0}{2022-09-27}{Replace 'master' for 'combined' and 'aggressor' for 'source'}{Andrés A. Plazas Malagón}
```

Comment:
Can you switch out "for" with "with" here?  My initial read had reversed the direction of the replacements.